Python/ds/stack_via_queue.py

```diff
@@ -0,0 +1,48 @@
+# Program to implement a stack using  
```

Comment:
@sonukushwaha403 similar to this https://www.geeksforgeeks.org/implement-stack-using-queue/